src/logic/tokens/utils/tokenHelpers.ts

```diff
@@ -14,7 +14,7 @@ import { TxServiceModel } from 'src/routes/safe/store/actions/transactions/fetch
 export const ETH_ADDRESS = '0x000'
 export const SAFE_TRANSFER_FROM_WITHOUT_DATA_HASH = '42842e0e'
 
-export const getEthAsToken = (balance: string): Token => {
+export const getEthAsToken = (balance: string | number): Token => {
```

Comment:
would it possible to unify the balances type across the app?

Reply:
oh, for this one I definitely will go with "let's solve it in a separate PR"